panel_of_normals/combine_variants.cwl

```diff
@@ -2,14 +2,16 @@
 
 cwlVersion: v1.0
 class: CommandLineTool
-label: "CombineVariants (GATK 3.6)"
-baseCommand: ["/usr/local/bin/jdk1.8.0_45/bin/java", "-jar", "/usr/local/bin/GATK3.6/GenomeAnalysisTK.jar", "-T", "CombineVariants"]
+label: "CombineVariants for Panel of Normals"
+baseCommand: ["/usr/bin/java", "-Xmx8g", "-jar", "/opt/GenomeAnalysisTK.jar", "-T", "CombineVariants"]
 requirements:
-    - class: DockerRequirement
-      dockerPull: "dbmi/gatk-docker:v1" #GATK 3.6 at a specific container revision
+    - class: ResourceRequirement
+      ramMin: 8000
+      tmpdirMin: 25000
 arguments:
-    ["-setKey", "null",
-     "--filteredAreUncalled", 
+    ["-minN", "2",
```

Comment:
I think it's fine for now to use a minimum number of samples equal to 2. Depending on the number of input VCFs, this may need to change in the future.